Update-5-request-payment-details.txt (#14)
degeri/Decred-Bug-Bounty-Proposal: 2019/email-templates/5-request-payment-details.txt

Subject: Bug Bounty Claim instructions for [email] - [(PREVMONTH)MM]/[(PREVMONTHYEAR)YYYY]

First time user:
    We are happy to inform you are now eligible for payment.

{LOOP AS NEEDED}
    You are eligible for a payment of <XXX> USD in DCR for issue {Link to issue}.
{END LOOP}

    {if non pgp: Please note payment was reduced by X USD due to your email not being encrypted with PGP]

>     If you would like to be listed on our hall of fame, please provide us with the name you would like to appear and you may choose to provide one link to your social media, github or blog.  

Reviewer: ```suggestion
    If you would like to be listed on our hall of fame, please provide us with the name you would like to appear and you may choose to provide one link to your social media, GitHub or blog.  
```

>     You will received an email with the subject "Welcome to the Contractor Management System" follow through on the link and register an account within 24 hours of receiving the email. 

Reviewer: ```suggestion
    You will receive an email with the subject "Welcome to the Contractor Management System" follow through on the link and register an account within 24 hours of receiving the email. 
```

>     Then click on "submit invoice" 

Reviewer: ```suggestion
    Then click on "Submit Invoice" 
```

    Month should be <MONTH> and Year should be <YEAR>. 

    "Contractor name"=[Full Name]
    "Contractor location" = [Your Location]
    "Contractor contact" = [Your email]
    "Contractor rate (USD)" = 5
    Payment address = [Your Decred Address]
    
     In the sheet:

{LOOP AS NEEDED}
    Type=2
    Domain=Development
    subdomain=Bug Bounty
    Description=[Link the github issue]
    Proposal Token=d33a2667469b56942adf42453def6cc2292325251e4cf791e806939ea9efc9e1
    Expense (USD)=<XXX>
{End LOOP}

    The total displayed should be {TOTAL USD}

    Once its filled hit "Submit"

>     You can then head over to https://cms.decred.org/user/invoices and ensure that your invoice has been submitted.Your payment will be made by the end of the current month.

Reviewer: ```suggestion
    You can then head over to https://cms.decred.org/user/invoices and ensure that your invoice has been submitted. Your payment will be made by the end of the current month.
```

>     Please note this website is for production use. Do not attempt test/find vulnerabilities or try to submit false/test invoices. Any such action might lead to suspension of your account.

Reviewer: ```suggestion
    Please note this website is for production use. Do not attempt to test/find vulnerabilities or try to submit false/test invoices. Any such action might lead to suspension of your account.
```

    If you need support creating a Decred wallet or address, you can visit our documentation website or you can can ask for help on our support channels.

    https://docs.decred.org/wallets/decrediton/decrediton-setup/
    https://decred.org/community/

    The Decred Project.

Registered user:

    We are happy to inform you are now eligible for payment.

{LOOP AS NEEDED}
    You are eligible for a payment of <XXX> USD in DCR for issue {Link to issue}.
{END LOOP}

    {if non pgp: Please note payment was reduced by X USD due to your email not being encrypted with PGP]

>     If you would like to be listed on our hall of fame, please provide us with the name you would like to appear and you may choose to provide one link to your social media, github or blog.  

Reviewer: ```suggestion
    If you would like to be listed on our hall of fame, please provide us with the name you would like to appear and you may choose to provide one link to your social media, GitHub or blog.  
```

>     You are alredy registed in the decred CMS system. Login to your account. 

Reviewer: ```suggestion
    You are already registered in the Decred CMS system. Login to your account. 
```

>     Then click on "submit invoice" 

Reviewer: ```suggestion
    Then click on "Submit Invoice" 
```

    Month should be <MONTH> and Year should be <YEAR>. 

    "Contractor name"=[Full Name]
    "Contractor location" = [Your Location]
    "Contractor contact" = [Your email]
    "Contractor rate (USD)" = 5
    Payment address = [Your Decred Address]
    
    In the sheet:

{LOOP AS NEEDED}
    Type=2
    Domain=Development
    subdomain=Bug Bounty
    Description=[Link the github issue]
    Proposal Token=d33a2667469b56942adf42453def6cc2292325251e4cf791e806939ea9efc9e1
    Expense (USD)=<XXX>
{End LOOP}

    The total displayed should be {TOTAL USD}

    Once its filled hit "Submit"

    You can then head over to https://cms.decred.org/user/invoices and ensure that your invoice has been submitted. Your payment will be made by the end of the current month.

>     Please note this website is for production use. Do not attempt test/find vulnerabilities or try to submit false/test invoices. Any such action will lead to suspension of your account.

Reviewer: ```suggestion
    Please note this website is for production use. Do not attempt to test/find vulnerabilities or try to submit false/test invoices. Any such action will lead to suspension of your account.
```
